examples/gltf-object-mapbox/layers.js

```diff
@@ -0,0 +1,72 @@
+import {PathLayer, ScatterplotLayer, TextLayer, PolygonLayer} from '@deck.gl/layers';
+
+const pathData =
+  'https://raw.githubusercontent.com/visgl/deck.gl-data/master/website/bart-lines.json';
+const stationData =
+  'https://raw.githubusercontent.com/visgl/deck.gl-data/master/website/bart-stations.json';
+const zipCodeData =
+  'https://raw.githubusercontent.com/visgl/deck.gl-data/master/website/sf-zipcodes.json';
+
+export const layers = [
+  new PolygonLayer({

```

Comment:
I'm not sure what all of these layers are doing. Looking at the filepath it has something to do with GLTF? Which a quick search says is related to 3d assets? Consider adding a README for both of these examples giving a brief overview of what they're showcasing.

Reply:
I'm deleting the GLTF example as it was from when we were learning.

Reply:
That's fine. The other examples could use a README though

Reply:
What do you think of the [README ](https://github.com/CodeLabs-Hubble-gl/hubble.gl/tree/hubble-example/examples/trips)file? I've never written one before so I'm not sure it's correct. The other examples didn't have readme files, should I add to those too?